planning/behavior_velocity_planner/src/scene_module/occlusion_spot/occlusion_spot_utils.cpp

```diff
@@ -317,11 +318,17 @@ PossibleCollisionInfo calculateCollisionPathPointFromOcclusionSpot(
    *             ------------------
    */
   PossibleCollisionInfo pc;
-  pc.arc_lane_dist_at_collision = arc_coord_occlusion_with_offset;
+  const double ttc = std::abs(arc_coord_occlusion_with_offset.distance / param.pedestrian_vel);
+  SafeMotion sm = calculateSafeMotion(param.v, ttc);
+  double distance_to_stop = arc_coord_occlusion_with_offset.length - sm.stop_dist;
+  const double eps = 1e-3;
+  if (distance_to_stop <= param.baselink_to_front) distance_to_stop = param.baselink_to_front + eps;
```

Comment:
if margin before stopping will become before path start point then it will be located from path start to baselink to front
![image](https://user-images.githubusercontent.com/65527974/152913379-8f02b563-717f-4589-8af2-a7b60aff09ef.png)

Reply:
21098c6 includes this change

![image](https://user-images.githubusercontent.com/65527974/152917375-d06ed928-244d-4af7-914f-22e67ee7e542.png)

Reply:
minor change to show debug collision point better(from start to end)
![image](https://user-images.githubusercontent.com/65527974/152949117-de76ab27-5018-42e6-bb86-fb91faf3ef51.png)